specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1088,9 +1088,78 @@
           }
         }
       }
+    },
+    "/randomnumbers": {
+      "post": {
+        "tags": [
+          "RandomNumbers"
+        ],
+        "operationId": "randomnumbers",
+        "summary": "Get the requested number of bytes containing random values.",
+        "description": "Get the requested number of bytes containing random values.",
```

Comment:
You might explain from where the random numbers come, i.e. a managed HSM. I can otherwise see questions like, "Why would Microsoft provide this when a local crypto RNG is likely available?"